jaxrs/src/main/java/org/killbill/billing/jaxrs/json/BulkBaseSubscriptionAndAddOnsJson.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2016 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.jaxrs.json;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+
+public class BulkBaseSubscriptionAndAddOnsJson {
+
+    @ApiModelProperty(required = true)
+    private final List<SubscriptionJson> baseEntitlementAndAddOns;
+
```

Comment:
@sbrossie 
So, I've been working on this part of the issue, modifying the `getAccountBundles()` Resource to accept the list of UUIDs as a `List<String>` called `bundlesFilter`.
The thing is that I had to modify the `buildResponse()` method at `JaxrsUriBuilder` in order to accept a `Map<String, Object>` of queryParams, and the problem is that at the end I'll have to call the endpoint like this:
```
http://127.0.0.1:8080/1.0/kb/accounts/ff5025cc-1026-4992-bc20-72b76d617e48/bundles?bundlesFilter=11c0ce4d-600d-4c06-9172-6ac36760679a&bundlesFilter=2a7d7916-fec2-4366-93ff-1c6627cfcb6e
```

As you can see, the `bundlesFilter` query parameter needs to be repeated so it can be read as a `List<String>` from the Resource.
That works, but I'm having some troubles when I try to "build" the query params Map that will be sent as part of the Location in the Response, because the Key is the same for all of them. I mean, I can do that but I'll have to create a new method in `JaxrsUriBuilder` ONLY for this case. 
Is that what you are expecting? Or should I just return the list of bundles for that Account?

Thanks!